lib/runtime/newWasmer/memory.go

```diff
@@ -0,0 +1,38 @@
+// Copyright 2023 ChainSafe Systems (ON)
+// SPDX-License-Identifier: LGPL-3.0-only
+
+package newWasmer
+
+import (
+	"errors"
+	"fmt"
+
+	wasmgo "github.com/wasmerio/wasmer-go/wasmer"
```

Comment:
```suggestion
	"github.com/wasmerio/wasmer-go/wasmer"
```
should be possible to just use wasmer